fieldC/checkOnAxis.c

```diff
@@ -0,0 +1,15 @@
+#include "field.h"
+
+int
+checkOnAxis(struct nodeEntry *pointsAndNodes, int numNodes)
+{
+int i, found = 0;
+
+	for (i = 0; i < numNodes; i++)
+		if (pointsAndNodes[i].x == 0 && pointsAndNodes[i].y == 0) {
+			found = 1;
```

Comment:
We should try to add some unit tests to validate the function of all of these things... especially since the majority of students in the lab are *not* as familiar w/ C-syntax, things can become much more susceptible to innocent bug introduction w/ syntax issues.

Reply:
are you suggesting that someone might modify my code? hmmmph.

maybe I'll work on some tests while I'm waiting to understand xdc_get; I'll see what I can come up with.

Reply:
Nope - additional code that runs yours to test it against some known outcomes... but then in the future when folks may dabble with changing the code, they have some reference tests that should still pass.